salt/metalk8s/addons/ui/deployed/ui.sls

```diff
@@ -59,4 +59,4 @@ Create ui-branding ConfigMap:
           namespace: metalk8s-ui
         data:
           theme.json: |
-           {"theme": {"light":{ "brand":{ "base": "#F7F6F9","baseContrast1": "#ECF4FF","primary": "#313B44","secondary": "#e99121","success": "#23AD56","info": "#00B2A9", "warning": "#F1B434","danger": "#EF3340","background": "#ffffff","backgroundContrast1": "#F7F6F9","backgroundContrast2": "#FAF9FB","text":"#000000","border":"#000000"},"logo_path":"/brand/assets/branding-light.svg"},"dark":{"brand":{"base": "#121214", "baseContrast1": "#192A41","primary": "#A7B6C3","secondary": "#2979ff","success": "#23AD56","info": "#00B2A9","warning": "#F1B434","danger": "#EF3340","background": "#0D0D0D","backgroundContrast1": "#1D1D21","backgroundContrast2": "#171717","text": "#ffffff","border": "#ffffff"},"logo_path":"/brand/assets/branding-dark.svg"},"custom":{}},"default": "light"}
+           {"theme": {"light":{"brand":{"base": "#607080", "primary": "#FAF9FB", "primaryDark1": "#F7F6F9", "primaryDark2": "#EDEAF0", "secondary": "#037AFF", "secondaryDark1": "#1C3D59", "secondaryDark2": "#1C2E3F", "success": "#006F62", "healthy": "#25AC56", "healthyLight": "#75FE63", "info": "#027AFF", "warning": "#FFC10A", "danger": "#EF3340", "critical": "#BE2543", "background": "#ffffff", "backgroundBluer": "#ECF4FF", "textPrimary": "#313B44", "textSecondary": "#8593A0", "textTertiary": "#6C8398", "borderLight": "#A5A5A5", "border": "#A5A5A5"},"logo_path": "/brand/assets/branding-light.svg"},"dark": {"base": "#6A7B92", "primary": "#1D1D1F", "primaryDark1": "#171718", "primaryDark2": "#0A0A0A", "secondary": "#037AFF", "secondaryDark1": "#1C3D59", "secondaryDark2": "#1C2E3F", "success": "#006F62", "healthy": "#25AC56", "healthyLight": "#75FE63", "info": "#027AFF", "warning": "#FFC10A", "danger": "#EF3340", "critical": "#BE2543", "background": "#121214", "backgroundBluer": "#182A41", "textPrimary": "#FFFFFF", "textSecondary": "#A8B5C1", "textTertiary": "#A7B6C3", "borderLight": "#2C3137", "border": "#A5A5A5"},"logo_path":"/brand/assets/branding-dark.svg"},"custom":{}},"default": "light"}
```

Comment:
Not exactly related to this PR, but having to duplicate the contents of `ui/public/brand/theme.json` in here seems unnecessary.

I get why we would want the ConfigMap to be in the cluster from the start: users would have a view of the defaults, and would be able to easily discover/edit their theme.
However, we should try to automatically keep it in sync with what we bake in the `metalk8s-ui` container. I think this can be done by templating in the buildchain (akin to what you did for Dex's UI IIRC), reading from `ui/public/...` directly.

Also, not sure how it's done today, but since we implement a fallback for these files in the container's `public` directory, we should handle the case where the ConfigMap does not exist (in the Deployment's Pod template).

All those items would need their own tickets, no need to make it land in this PR.